test/models/test_openai_model.py

```diff
@@ -23,9 +23,12 @@
 
 @pytest.mark.model_backend
 @pytest.mark.parametrize("model_type", [
-    ModelType.GPT_3_5_TURBO,
-    ModelType.GPT_3_5_TURBO_16K,
     ModelType.GPT_4,
+    ModelType.GPT_4_32k,
+    ModelType.GPT_4_TURBO,
+    ModelType.GPT_4_TURBO_VISION,
+    ModelType.GPT_3_5_TURBO,
+    ModelType.GPT_3_5_TURBO_16k,
```

Comment:
Prefer moving the `GPT_3_5_TURBO` and `GPT_3_5_TURBO_16k` to the top.

Reply:
> Prefer moving the `GPT_3_5_TURBO` and `GPT_3_5_TURBO_16k` to the top.

done